readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
+
+## {Project Name} 
+{Optional: Include the project URL and the project owner name}
+
+## Table of contents 
+{This section is optional.}
+1. Heading 1
+2. Heading 2
+3. Heading 3...
+
+## Project description 
+{Project description example available in README template guide}
+
+With *{Project Name}* you can *(verb)* *(noun)*...
+
+*{Project Name}* helps you *(verb)* *(noun)*....
+
+Unlike *(alternative)*, *{Project Name}* *(verb)* *(noun)*... 
+
+{Optional: Include screenshots and/or demo videos}
+
+## Who this project is for 
+This project is intended for {target audience} who want to {audience objective}. 
+
+## Project dependencies
+To use {Project Name}, you will need: 
+- Prerequisite 1 
+- Prerequisite 2
+- Prerequisite 3...
+
+## Instructions for using {Project Name}
+Get started with {Project Name} by {first step here}
+
+### How to get {Project Name}
+1. {Write the step here. Use a verb to start.}
+
+    {Explanatory text here}
+
+    {Optional: Code sample or screenshot that helps your users complete this step}
+
+2. {Write the step here. Use a verb to start.}
+
+    a. {Substep 1}
+    
+    b. {Substep 2}
+
+### How to install {Project Name}
+1. {Write the step here. Use a verb to start.}
+
+2. {Write the step here. Use a verb to start.}
+### How to configure {Project Name}
```

Comment:
**nitpick:** I think there's an empty line missing between the lines
- 56 and 57
- 60 and 61
- 64 and 65